core/arch/x86_64/x86_64.mk

```diff
@@ -37,23 +37,34 @@ endif
 
 core-platform-cppflags	+= -I$(arch-dir)/include
 core-platform-subdirs += \
-	$(addprefix $(arch-dir)/, kernel tee mm) $(platform-dir)
+	$(addprefix $(arch-dir)/, kernel mm tee) $(platform-dir)
 
 core-platform-subdirs += $(arch-dir)/sm
 
+platform-cflags-generic ?= -ffunction-sections -fdata-sections -pipe
+platform-aflags-generic ?= -pipe
+platform-aflags-generic += -D__ASSEMBLY__
 x86-64-platform-cppflags += -DX86_64=1 -D__LP64__=1 -DLTC_NO_ROLC -DLTC_NO_FAST
```

Comment:
Please investigate if we can remove "-DLTC_NO_ROLC -DLTC_NO_FAST".

Reply:
the 2 settings are only for tomcrypt library, we can remove them